internal/auth/options.go

```diff
@@ -93,10 +98,13 @@ type Options struct {
 	SetXAuthRequest    bool `envconfig:"SET_XAUTHREQUEST" default:"false"`
 
 	// These options allow for other providers besides Google, with potential overrides.
-	Provider       string `envconfig:"PROVIDER" default:"google"`
+	Provider string `envconfig:"PROVIDER" default:"google"`
+
 	SignInURL      string `envconfig:"SIGNIN_URL"`
 	RedeemURL      string `envconfig:"REDEEM_URL"`
+	RevokeURL      string `envconfig:"REVOKE_URL"`
 	ProfileURL     string `envconfig:"PROFILE_URL"`
+	UserInfoURL    string `envconfig:"USERINFO_URL"`
```

Comment:
Is this different than `ProfileURL`? If so, what is the difference?

Reply:
Interesting! It hadn't really triggered that it was defined but not in use at all within the `auth` configs, only the `proxy` configs. I think yes, it makes sense for this to be used in place of `UserInfoURL`! I've removed.